audio-worklet/src/bitcrusher-demo.js

```diff
@@ -85,16 +100,49 @@ class BitcrusherDemo {
           default: this.masterGain_.gain.value,
           name: 'Volume'
         });
+
+    // GUI controls allow switching between audio worklet and script processor.
+    // The audio worklet node is used by default.
+    this.useAudioWorkletCheckBox_ = document.createElement('input');
+    this.useAudioWorkletCheckBox_.type = 'checkbox';
+    this.useAudioWorkletCheckBox_.checked = true;
+    const audioworkletLabel = document.createElement('label');
+    audioworkletLabel.textContent =
+        'Use AudioWorklet (if unchecked, use ScriptProcessor Bitcrusher).';
+
+    this.useAudioWorkletCheckBox_.addEventListener(
+        'change', this.switchBitcrusherNode.bind(this));
+
+    let container = document.getElementById(containerId);
+    container.appendChild(this.useAudioWorkletCheckBox_);
+    container.appendChild(audioworkletLabel);
+  }
+
+  /**
+   * Switch between whether the audio worklet node or script processor node
+   * connects to the master gain node.
+   */
+  switchBitcrusherNode() {
+    // The function is called after the checkbox switches state.
+    const changeToScriptProcessor = !this.useAudioWorkletCheckBox_.checked;
```

Comment:
It's inverted since the checkbox will have unchecked itself before the switchBitcrusherNode is called from the event listener.